pwa/src/components/componentCard/ComponentCard.module.css

```diff
@@ -0,0 +1,92 @@
+.container {
```

Comment:
The margins between the tags may all be equal (horizontal and vertical).

Reply:
![image](https://user-images.githubusercontent.com/66552932/185345704-d48a197d-916b-4f2e-a359-15c67b8c5c5a.png)

Reply:
Note: also increased the margin between the tags and the description.